compio-driver/src/iour/mod.rs

```diff
@@ -97,9 +97,21 @@ impl Driver {
         trace!("new iour driver");
         let notifier = Notifier::new()?;
         let mut io_uring_builder = IoUring::builder();
-        if let Some(sqpoll_idle) = builder.sqpoll_idle {
-            io_uring_builder.setup_sqpoll(sqpoll_idle.as_millis() as _);
+
+        match builder.sqpoll_idle {
+            Some(sqpoll_idle) => {
+                io_uring_builder.setup_sqpoll(sqpoll_idle.as_millis() as _);
+            }
+
+            None => {
```

Comment:
Is it better to add an option to the builder?

Reply:
you means let user to choose enable the `coop_taskrun` or not?

Reply:
Yes.

Reply:
if add this option, how to handle the conflict between `coop_taskrun` and `sqpoll`
1. when setup builder, panic if both are set
2. set method return `Result`
3. ignore `sqpoll` when set `coop_taskrun`

Reply:
it does nothing, just pass them all to syscall and let syscall return invalid argument

Reply:
Then. I think it's OK to not consider about that.

Reply:
ok